### PR TITLE
add trial adjusted dprime to make_daily_figure

### PR DESCRIPTION
**Pull request by corbennett**
@dougollerenshaw I've modified make_daily_figure and make_rolling_dprime_plot to display both the adjusted and legacy dprime values. Let me know if this looks ok.

### CODE REVIEW

File: visual_behavior/visualization/extended_trials/daily.py [hunk 1]
```diff
@@ -323,12 +328,19 @@ def make_daily_figure(
...
+        hit_rate, fa_rate, d_prime = get_response_rates(
+                df_nonaborted,
+                sliding_window=sliding_window,
+                apply_trial_number_limit=apply_trial_number_limit
+                )
+        peak_dprime = session_metrics.peak_dprime(extended_trials, first_valid_trial=first_valid_trial, apply_trial_number_limit=apply_trial_number_limit)
+        if not np.isnan(peak_dprime):
+            make_rolling_dprime_plot(d_prime, ax[3], peak_dprime=peak_dprime, append_ticks=append_ticks, color=color)
+        else:
+            make_rolling_dprime_plot(d_prime, ax[3], append_ticks=append_ticks, color=color)
```
Comment (dougollerenshaw) [inline visual_behavior/visualization/extended_trials/daily.py:343]: Is it obvious which line belongs to which condition? It seems like a legend would be a good idea.

Comment (corbennett) [reply visual_behavior/visualization/extended_trials/daily.py]: Agreed. Done!

Comment (corbennett) [reply visual_behavior/visualization/extended_trials/daily.py]: @dougollerenshaw also, the linting errors that are being caught below aren't for any of the code I modified (or even in the same file). They are primarily here:

visual_behavior/visualization/qc/dash_app/app.py

I'll go ahead and fix them. Not totally sure how this happened...

**Comment by dougollerenshaw** [comment]
Thanks @corbennett! I'm also not sure why those linting errors would just now be showing up.

**Pull request merged**